### ISSUE TITLE
Simplify ocean global stats analysis member

### ISSUE THREAD

**Issue by mark-petersen** [MPAS-Dev/MPAS-Model#413]
The `global_stats` analysis member currently includes these scalar statistics for a large number of variables:

1. **Min** Minimum global value of salinity in ocean cells.
1. **Max** Maximum global value of salinity in ocean cells.
1. **Avg** Average value of salinity in ocean cells.
1. **Sum** Accumulated global value of salinity in ocean cells.
1. **Rms** Global root mean square value of salinity in ocean cells.
1. **MinVertSum** Minimum vertical sum of salinity in ocean cells.
1. **MaxVertSum** Maximum vertical sum of salinity in ocean cells.

I find the first four very useful, but I have never used RMS, MinVertSum, MaxVertSum. I propose that we remove these three, as the global stats calculation currently has 235 variables, and was slow enough to cause problems on some past simulations.

**Comment by mark-petersen** [comment]
@pwolfram @maltrud @milenaveneziani @vanroekel @xylar:

1. Would anyone mind if we remove RMS, MinVertSum, MaxVertSum from the global stats analysis member? For rare special cases these can be computed from netcdf output with nco commands.
1. Should we also remove Sum? It is useful for testing conservation, but can be computed from a restart file. Without surface fluxes, Avg tells us if a tracer is conserved, and `layerThicknessSum` gives the total volume.

**Comment by xylar** [comment]
Yep, I don't have a problem with removing RMS, MinVertSum and MaxVertSum.

I don't think we can eliminate Sum because I think some of the quantities are needed for a full accounting of conservation within E3SM, though I'm not 100% clear on how this is being tracked in practice.  @vanroekel, any thoughts?

**Comment by vanroekel** [comment]
@xylar  you are right.  When we were tracking down the water conservation issues the Sum variables were quite useful, but there is no formal analysis that uses those.  I think it is okay to compute conservation from a restart file as @mark-petersen suggests.  I have never used the other variables.

But may I ask why there is a need to do this?  The computational cost and IO from this has to be minute.  It doesn't seem pressing to me, but I may be missing something.

**Comment by mark-petersen** [comment]
@vanroekel you are right, it is not pressing. The motivation is that I use the global stats a great deal in stand-alone mode, and 235 fields in global stats is difficult to navigate and not intuitive to students and collaborators. I do remember that global stats was so slow that it had to be turned off for some past simulations, but that have been the pnetcdf configuration.

I am writing up a few of these issues because they are good small introductory projects, and contribute to a cleaner code base.
